diff -up jruby-9.2.0.0/lib/ruby/stdlib/rubygems/defaults/jruby.rb.omv~ jruby-9.2.0.0/lib/ruby/stdlib/rubygems/defaults/jruby.rb
--- jruby-9.2.0.0/lib/ruby/stdlib/rubygems/defaults/jruby.rb.omv~	2018-06-09 23:40:22.058955752 +0200
+++ jruby-9.2.0.0/lib/ruby/stdlib/rubygems/defaults/jruby.rb	2018-06-09 23:40:42.644956880 +0200
@@ -24,25 +24,6 @@ module Gem
     }
   end
 
-  # Default home directory path to be used if an alternate value is not
-  # specified in the environment.
-  #
-  # JRuby: We don't want gems installed in lib/jruby/gems, but rather
-  # to preserve the old location: lib/ruby/gems.
-  def self.default_dir
-    dir = RbConfig::CONFIG["default_gem_home"]
-    dir ||= File.join(ConfigMap[:libdir], 'ruby', 'gems', 'shared')
-    dir
-  end
-
-  # Default locations for RubyGems' .rb and bin files
-  def self.default_rubygems_dirs
-    [
-        File.join(ConfigMap[:libdir], 'ruby', 'stdlib'),
-        ConfigMap[:bindir]
-    ]
-  end
-
   ##
   # Is this a windows platform?
   #
