diff --git a/bin/jruby.bash b/bin/jruby.bash
index ff1c532..e61198c 100755
--- a/bin/jruby.bash
+++ b/bin/jruby.bash
@@ -150,6 +150,35 @@ if $cygwin; then
     JRUBY_CP=`cygpath -p -w "$JRUBY_CP"`
 fi
 
+j=`build-classpath \
+    objectweb-asm/asm \
+    objectweb-asm/asm-commons \
+    objectweb-asm/asm-tree \
+    objectweb-asm/asm-analysis \
+    objectweb-asm/asm-util \
+    joni jnr-netdb/jnr-netdb \
+    jnr-enxio \
+    jnr-x86asm \
+    jnr-unixsocket/jnr-unixsocket \
+    jnr-posix \
+    bytelist \
+    jnr-constants \
+    jcodings \
+    jnr-ffi/jnr-ffi \
+    jffi/jffi \
+    jffi/jffi-native \
+    snakeyaml \
+    commons-codec \
+    base64coder \
+    jzlib \
+    invokebinder \
+    nailgun/nailgun-server \
+    yecht/yecht \
+    joda-time \
+    options/options`
+
+JRUBY_CP="$JRUBY_CP$CP_DELIMITER$j"
+
 # ----- Set Up The System Classpath -------------------------------------------
 
 if [ "$JRUBY_PARENT_CLASSPATH" != "" ]; then
